flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -40,6 +40,14 @@ under the License.
 		<reflections.version>0.9.8</reflections.version>
 	</properties>
 
+	<repositories>
+		<repository>
+			<!-- Needed for org.pentaho:pentaho-aggdesigner-algorithm:pom:5.1.5-jhyde -->
+			<id>spring-lib-release</id>
+			<url>https://repo.spring.io/libs-release/</url>
```

Comment:
according to https://mvnrepository.com/artifact/org.pentaho/pentaho-aggdesigner-algorithm/5.1.5-jhyde

> Note: this artifact it located at Spring Plugins repository (https://repo.spring.io/plugins-release/)

Reply:
Can we exclude pentaho like jms?

Reply:
not sure, what's the usage of this pentaho?

Reply:
This is dependency,  not a plugin. So this repo url is strange.  
I'm not sure what the purpose of this library is. I have seen multiple projects where this exact dependency has been causing problems. 

If there is a better way to fix this I'll gladly refactor my pull request

Reply:
Try to add exclude to hive-core?
```
<exclusion>
    <groupId>org.pentaho</groupId>
    <artifactId>pentaho-aggdesigner-algorithm</artifactId>
</exclusion>
```